src/test/golden.ts

```diff
@@ -128,6 +157,12 @@ export function compare(actual: Buffer | string, name: string, snapshotPath: (na
   if (result.diff)
     fs.writeFileSync(diffPath, result.diff);
 
+  if (withNegateComparison) {
```

Comment:
I think we should skip the block above (lines 151-158) that writes actual/expected/diff in this case, because everything worked out as expected (snapshot did not match), so we don't need extra artifacts.

Reply:
I've decided to return in because it may be usefull to understand what's going on. And if needed take an expected image from the `test-results` and update snapshot manually.  
But **I agree with you** because I personally find artifacts for passing tests quite annoying.  
Maybe we should keep them with another suffix?

Reply:
I'd suggest to not write these files for now, and then gather feedback from the users first. 

Do they miss details in this case? From what I see we'll never print the path to the actual/expected/diff in this case, so 99% won't know where to look.